Makefile

```diff
@@ -417,6 +417,7 @@ tests/%.parse: tests/%
 	rm -rf $@-out
 
 tests/specs/functional/%.prove: TEST_SYMBOLIC_BACKEND=haskell
+tests/specs/functional/storageRoot-spec.k.prove: TEST_SYMBOLIC_BACKEND=java
```

Comment:
This is here because I couldn't get the haskell backend to finish the proof. It seems like it's getting stuck on my `#intMap2StorageMap` function.

Reply:
I'm going to leave it like this for the time being. There are multiple places in the Merkle Tree semantics that iterate over Map keys and I don't want to change all of them only to change them back once the haskell backend supports it.